Ars_Magica_5th/sheet.json

```diff
@@ -1,8 +1,9 @@
 {
     "html": "Ars_Magica_5th.html",
     "css": "Ars_Magica_5th.css",
-    "authors": "Medieve & Felan, original sheet by Coal Powered Puppet.",
+    "authors": "Medieve (Richard T.), original sheet by Coal Powered Puppet.",
+    "Patreon": "https://www.patreon.com/sceneficant",
```

Comment:
Patreon is capitalized here but it needs to be lower-case for it to work as expected